requirements.txt

```diff
@@ -1,3 +1,4 @@
 ops ~= 2.5
 jinja2
 jsonschema
+pydantic
```

Comment:
this dep needs some version constraints so we don't get breaking changes surprises